PORT-3606-docs-fix-advanced-form-configurations-page (#306)
port-labs/port-docs: docs/create-self-service-experiences/setup-ui-for-action/advanced-form-configurations.md

![Cluster And Namespace Action](../../../static/img/software-catalog/blueprint/javascriptSDK.png)

### Filter the dropdown's available options based on a property

> This example contains a filter that will only display only the envs that are not with role "production".

Reviewer: ```suggestion
This example contains a filter that will only display environment entities whose type is not `production`:
```

```json
{
  "env": {
    "type": "string",
    "format": "entity",

>     "blueprint": "env",

Reviewer: ```suggestion
    "blueprint": "environment",
```

    "dataset": {
      "combinator": "and",
      "rules": [
        {

>           "property": "role",

Reviewer: ```suggestion
          "property": "type",
```

          "operator": "!=",
          "value": "production"
        }
      ]
    }
  }
}
```

![Only Production Envs](../../../static/img/software-catalog/blueprint/onlyNotProductionEnvs.png)

> :point_up: only the environments which are not with the role "production" will appear in the dropdown. :point_up:

Reviewer: ```suggestion
:point_up: only the environments whose type is not `production` will appear in the dropdown. :point_up:
```
